### PR TITLE
DeploymentOptions model updates according to STS model changes

### PR DESCRIPTION
**Pull request by ssreerama**
These changes are in sync to STS deploymentOptions model changes which have been added new options and changed the model to support options values, description and displayNames. Thanks.

Pending work: Need to update STS version after STS changes are checked in.

### CODE REVIEW

File: typings/vscode-mssql.d.ts [hunk 1]
```diff
@@ -489,85 +489,112 @@ declare module 'vscode-mssql' {
 		executeAndScript = 2
 	}
 
+	export interface DacDeployOptionPropertyBoolean {
```
Comment (kisantia) [inline typings/vscode-mssql.d.ts:492]: please add doc comments for describing what these interfaces are for

**Review by kisantia — APPROVED** [review]
Please make sure to also update the STS version in this PR before merging.

File: typings/vscode-mssql.d.ts (hunk 1 above)
Comment (ssreerama) [reply typings/vscode-mssql.d.ts:495]: done

**Comment by ssreerama** [comment]
Verified the publish database with load profile (created from Visual studio) and publish database is succeeded from the mssql extension from VSCode.

**Pull request merged**